src/Altinn.Platform/Altinn.Platform.Storage/Storage/appsettings.json

```diff
@@ -25,6 +25,7 @@
     "BlobEndPoint": "http://127.0.0.1:10000/devstoreaccount1"
   },
   "GeneralSettings": {
-    "BridgeRegisterApiEndpoint": "https://at21.altinn.cloud/sblbridge/register/api/"
+    "OpenIdWellKnownEndpoint": "https://platform.at21.altinn.cloud/authentication/api/v1/openid/",
```

Comment:
hardcoding the environment? :O

Reply:
Could we replace with localhost:5040 ?

Reply:
The property "OpenIdWellKnownEndpoint" will be replaced by Chart values and we can of course change it to localhost:5040. :)